diff -Naur mariadb-connector-odbc-3.0.1-beta-src/CMakeLists.txt mariadb-connector-odbc-3.0.1-beta-src-new/CMakeLists.txt
--- mariadb-connector-odbc-3.0.1-beta-src/CMakeLists.txt	2017-08-01 14:24:14.000000000 +0200
+++ mariadb-connector-odbc-3.0.1-beta-src-new/CMakeLists.txt	2017-09-04 13:53:51.140537112 +0200
@@ -230,6 +230,10 @@
                ${CMAKE_SOURCE_DIR}/mariadb-odbc-driver-uni.def)
 
 ####### MAODBC #######             
+IF(MARIADB_DYNAMIC_LIB)
+  SET(MARIADB_LIB ${MARIADB_DYNAMIC_LIB})
+ENDIF()
+
 ADD_LIBRARY(${LIBRARY_NAME} SHARED ${MARIADB_ODBC_SOURCES} ${CMAKE_SOURCE_DIR}/mariadb-odbc-driver-uni.def maodbcu.rc)
 
 SET_TARGET_PROPERTIES(maodbc PROPERTIES LANGUAGE C)
